simphony/cuds/lattice.py

```diff
@@ -67,6 +69,43 @@ def origin(self):
         return self._origin
 
     def get_node(self, index):
+        pass
```

Comment:
I think it's necessary to implement these so that BaseLattice class can be instantiated. Instantiation is needed in using H5CUDS.add_lattice with a BaseLattice parameter that has all the "metadata" included, but no DataContainer data. Also, we think that BaseLattice will be used as a common ancestor for all future XLattice classes. 

In the case of FileLattice, typical lattice data (used in actual simulation) does not fit into memory of a single pc. However, it is useful to allow adding of a FileLattice to a H5CUDS-file by using just a single computer instead of a computer cluster where the full lattice data could fit into. Using a common ancestor here keeps the required parameters add_lattice nice and clean, and it still works as before.